pylint/checkers/design_analysis.py

```diff
@@ -81,6 +86,18 @@
         "too-many-boolean-expressions",
         "Used when an if statement contains too many boolean expressions.",
     ),
+    "R0917": (
+        "Method(s) defined on multiple direct parents without reimplementation: %s",
+        "order-dependent-resolution",
+        "Used when methods defined on multiple direct parents are not reimplemented, "
+        "leading to fragility where the order of parents dictates behavior.",
+    ),
+    "R0918": (
+        "super() called despite multiple direct parents defining %s",
```

Comment:
```suggestion
        "super() called despite %s defining %s",
```
...despite 'HarshParent', 'SomeClass' and  'KindParent' defining... The more info we add to the actual message the less confusing it is for someone who don't initially know what order dependent resolution is.